docs/lesson03/prepare.md

```diff
@@ -95,17 +94,45 @@ caption="Debugger showing Function Stack"
 %}
 
 
+<!-- Python Version
 ### Stacks in Python
 
 In Python, a stack can be represented using a list. To push an item to the back of the stack, the append function can be used on the list. To pop items from the back of the stack, the pop function can be used. The pop function will also delete it from the list. The size can be determined by using the len function on the list. The performance of the stack using a Python list is based on the performance of the dynamic array.
 
 
-| **Common Stack Operation** | **Description**                                              | **Python Code**            | **Performance**                                                          |
+| **Common Stack Operation** | **Description**                                      | **Python Code**        | **Performance**                                                      |
 |------------------------|----------------------------------------------------------|------------------------|----------------------------------------------------------------------|
 | push(value)            | Adds "value" to the back of the stack.                   | my_stack.append(value) | ***O(1)*** - Performance of adding to the end of a dynamic array     |
 | pop()                  | Removes and returns the item from the back of the stack. | value = my_stack.pop() | ***O(1)*** - Performance of removing from the end of a dynamic array |
```

Comment:
We should probably change these to use the C# naming conventions, also a Stack in C# has Push and Pop as the method names, and Size as the property stack.Count